software/oort-config/role/firewall/etc/mosquitto/mosquitto.conf

```diff
@@ -0,0 +1,918 @@
+# Config file for mosquitto
+#
+# See mosquitto.conf(5) for more information.
+#
+# Default values are shown, uncomment to change.
+#
+# Use the # character to indicate a comment, but only if it is the
+# very first character on the line.
+
+# =================================================================
+# General configuration
+# =================================================================
+# Example usage from localhost only:
+# use steps in `man mosquitto-tls` to generate certs
+# 	mosquitto_sub -h localhost --cafile /etc/mosquitto/ca.crt -p 8883 -t test -d --insecure
+# mosquitto_pub -h localhost --cafile /etc/mosquitto/ca.crt -p 8883 -t test -d -m hello --insecure
+# 
+listener 8883
+cafile /etc/mosquitto/ca.crt
```

Comment:
Is it a reasonably good assumption that the user won't want to use mosquitto for anything other than this? Or does it make sense to create an "ohsnap" subdirectory here (or elsewhere)? I know this is just a proof of concept so maybe I'm overthinking.